conf/proxy.conf

```diff
@@ -60,6 +60,9 @@ advertisedAddress=
 # Enable or disable the HAProxy protocol.
 haProxyProtocolEnabled=false
 
+# Enable or disable the zero-copy of NIC to NIC.
+zeroCopyModeEnabled=true
```

Comment:
zero-copy can mean a lot of things. Can we be more explicit here ?

Reply:
ok, I will add more descriptions.

Reply:
> I would name this like 'proxyZeroCopyModeEnabled'

ok, the name is more clear